pkgs/development/tools/continuous-integration/woodpecker-cli/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  version = "0.15.1";
+  pname = "woodpecker-cli";
+  revision = "v${version}";
+
+  vendorSha256 = null;
+
+  doCheck = true;
+
+  src = fetchFromGitHub {
+    owner = "woodpecker-ci";
+    repo = "woodpecker";
+    rev = revision;
+    sha256 = "sha256-ilywzKczcER7kO19X6vhn28HieeT7ZTSsoKyu802bS0=";
+  };
+
+  buildPhase = ''
+    make build-cli
```

Comment:
Rather than rely on `make` you might be able to use `subPackages = [ "cmd/cli" ];` instead, and while I haven't tested it could resolve the failing to build on non-linux/amd64

Reply:
Didn't know about subPackages :D That's solves all the issues

Reply:
Also i tried with woodpecker-agent without issues, but with woodpecker-server (which need CGO and yarn) i got a bunch of errors :/